src/main/scala/com/mwz/sonar/scala/scoverage/ScoverageSensor.scala

```diff
@@ -40,39 +40,38 @@ final class ScoverageSensor(scoverageReportParser: ScoverageReportParserAPI) ext
   /** Populates the SensorDescriptor of this sensor. */
   override def describe(descriptor: SensorDescriptor): Unit = {
     descriptor
+      .name(SensorName)
       .onlyOnLanguage(Scala.LanguageKey)
       .onlyOnFileType(InputFile.Type.MAIN)
-      .name(SensorName)
   }
 
-  /** Saves in SonarQube the scoverage information of a module */
+  /** Saves in SonarQube the scoverage information of the project. */
   override def execute(context: SensorContext): Unit = {
     log.info("Initializing the scoverage sensor.")
     log.debug(s"The current working directory is: '${PathUtils.cwd}'.")
     val settings = context.config
     val filesystem = context.fileSystem
 
-    val modulePath = getModuleBaseDirectory(filesystem)
-    val reportPath = modulePath.resolve(getScoverageReportPath(settings))
+    val projectPath = getProjectBaseDirectory(filesystem)
+    val reportPath = projectPath.resolve(getScoverageReportPath(settings))
     val sources = Scala.getSourcesPaths(settings)
-    val sourcePrefixes = sources.map(PathUtils.relativize(PathUtils.cwd, modulePath, _))
+    val sourcePrefixes = sources.map(PathUtils.relativize(PathUtils.cwd, projectPath, _))
     log.debug(s"The source prefixes are: ${sourcePrefixes.mkString("[", ",", "]")}.")
     log.info(s"Loading the scoverage report file: '$reportPath'.")
 
-    Try(scoverageReportParser.parse(reportPath, modulePath, sourcePrefixes)) match {
-      case Success(moduleCoverage) =>
+    Try(scoverageReportParser.parse(reportPath, projectPath, sourcePrefixes)) match {
+      case Success(projectCoverage) =>
         log.info("Successfully loaded the scoverage report file.")
-        log.debug(s"Module scoverage information: $moduleCoverage.")
+        log.debug(s"Project scoverage information: $projectCoverage.")
 
-        saveComponentScoverage(context, context.module(), moduleCoverage.moduleScoverage)
+        saveComponentScoverage(context, context.project, projectCoverage.projectScoverage)
```

Comment:
You might still need to save this against the current module otherwise the scanner explodes.

Reply:
Uhm.. the problem is that `module` is deprecated and the replacement is `project`.
And, I am not talking about the big picture about module vs project analysis, but about the context methods...

And  I thought I understood  that using the old context would be enough...
Because for the plugin each **module** would be seen as a **project** and that the server will aggregate the metrics...

But, apparently, I was wrong.
Can you check if using the deprecated `module` solves this problem?
Other option would be to remove the metric for the module, and only register the files.

Reply:
`module` is deprecated but it doesn't mean that it doesn't work any more - it will be going away at some point but it still works fine in 7.6. I tested the changes on my branch [here](https://github.com/mwz/sonar-scala/compare/sonarqube-7.6?expand=1) and as you can see from the diff I didn't change any `module` to `project`.

Reply:
@mwz Oh yeah, I know it still works _(that is the propose of deprecating things anyways lol)_.

I mean that, IMO _(or what I understand from the docs)_, the idea was that if we used the old _(almost deprecated)_ `Plugin`, instead of the new `ProjectPlugin`, the `project` would be the `module`.

But apparently it does not. - And, since if you said it works, I will revert the changes.

Do you want me to revert it everywhere _(a couple of logs)_?, or just there?

Also, since the are no more **_"modules"_** at interface level, how does this metrics are displayed?
That is why I suggested just removing the saveMetrics step for the `modoule` / `project` _(or whatever)_ until this gets solved.

Reply:
> And, since if you said it works, I will revert the changes.
> 
> Do you want me to revert it everywhere _(a couple of logs)_?, or just there?

Great, thanks. I don't mind the logs that much, I'm ok to leave them as they are.

> Also, since the are no more **_"modules"_** at interface level, how are these metrics displayed?

I'm not sure what happens to those - last time I tested it I might have seen some warning regarding those - I'll check again. I though those contributed to the overall metrics for the whole projects but it might not be the case anymore.

Reply:
Ok. So for now, I will just revert the changes here.
Will commit and push in a couple of minutes.